scss/_navbar.scss

```diff
@@ -64,51 +96,56 @@
 //
 // Used for brand, project, or site names.
 
+// Boosted mod
 .navbar-brand {
-  // Boosted mod
   position: relative;
   display: inline-flex;
   align-items: flex-end;
-  align-self: center;
-  // End mod
+  order: -1;
   padding-top: var(--#{$prefix}navbar-brand-padding-y);
   padding-bottom: var(--#{$prefix}navbar-brand-padding-y);
-  margin: map-get($spacers, 2) var(--#{$prefix}navbar-brand-margin-end); // Boosted mod
+  margin-top: var(--#{$prefix}navbar-brand-margin-y);
+  margin-right: var(--#{$prefix}navbar-brand-margin-end);
+  margin-bottom: var(--#{$prefix}navbar-brand-margin-y);
   @include font-size(var(--#{$prefix}navbar-brand-font-size));
-  font-weight: $font-weight-bold; // Boosted mod
-  line-height: .75; // Boosted mod
   color: var(--#{$prefix}navbar-brand-color);
   text-decoration: if($link-decoration == none, null, none);
-
-  /* rtl:remove */
-  letter-spacing: inherit; // Boosted mod
   white-space: nowrap;
-  @include transition($navbar-brand-transition); // Boosted mod
-  @include target-size($navbar-brand-logo-height); // Boosted mod
+  @include transition($navbar-brand-transition);
 
   &:hover,
   &:focus {
     color: var(--#{$prefix}navbar-brand-hover-color);
     text-decoration: if($link-hover-decoration == underline, none, null);
   }
 
-  // Boosted mod
-  //// Ensure headings variant align consistently
-  h1,
-  [class*="display-"],
-  .h1, .h2, .h3, .h4, .h5, .h6 { // stylelint-disable-line selector-list-comma-newline-after
-    margin: 0 0 0 $spacer;
-    line-height: inherit;
-  }
-
   //// Orange's master logo
   img {
-    width: $navbar-brand-logo-minimized-height;
-    height: $navbar-brand-logo-minimized-height;
+    width: auto;
+    height: var(--#{$prefix}navbar-brand-logo-size);
     @include transition($navbar-brand-logo-transition);
   }
-  // End mod
+
+  //// Ensure headings variant align consistently
+  .title,
+  .two-lined {
+    margin: divide(-2em, 7) $spacer divide(-1em, 7) calc(var(--#{$prefix}navbar-brand-logo-size) / 2); /* stylelint-disable-line function-disallowed-list */
```

Comment:
I observe that the title (on 1 line) is slighty off (1 or 2px). If I change this line to `margin: divide(-2em, 7) $spacer -4px calc(var(--#{$prefix}navbar-brand-logo-size) / 2); /* stylelint-disable-line function-disallowed-list */` it fixes the alignment (on my side) of the title on 1 line but breaks the title on 2 lines.

Reply:
More info about this issue.

**Ubuntu > Chrome**
* :green_circle: ![Screenshot from 2022-04-05 12-40-21](https://user-images.githubusercontent.com/17381666/161737162-97fed43c-3861-4f69-96a6-754838572623.png)

* :red_circle: ![Screenshot from 2022-04-05 12-42-14](https://user-images.githubusercontent.com/17381666/161737161-768d34a1-7bdf-4bd3-8fd1-d4132fe48a62.png)

**Ubuntu > Firefox**
* :red_circle: ![Screenshot from 2022-04-05 12-45-13](https://user-images.githubusercontent.com/17381666/161737504-2aa551c2-5074-43c1-9a36-b5a10363f0b4.png)
* :red_circle: ![Screenshot from 2022-04-05 12-44-48](https://user-images.githubusercontent.com/17381666/161737506-2e26fa7d-703c-4b16-8221-ea62162135e7.png)